test/operators/update-fee.ts

```diff
@@ -0,0 +1,183 @@
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).executeOperatorFee(1
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  it('Execute declared fee without a pending request reverts "NoPendingFeeChangeRequest"', async () => {
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('NoPendingFeeChangeRequest');
+  });
+
+  it('Execute declared fee too early reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod - 10);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  it('Execute declared fee too late reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod + helpers.CONFIG.executeOperatorFeePeriod + 1);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  //Dao
+  it('DAO get fee increase limit', async () => {
+    expect(await ssvNetworkContract.getOperatorFeeIncreaseLimit()).to.equal(helpers.CONFIG.operatorMaxFeeIncrease);
+  });
+
+  it('DAO get declared fee', async () => {
+    const newFee = initialFee + initialFee / 10;
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, newFee), [GasGroup.REGISTER_OPERATOR]);
+    const [feeDeclaredInContract] = await ssvNetworkContract.getOperatorDeclaredFee(1);
+    expect(feeDeclaredInContract).to.equal(newFee);
+  });
+
+  it('DAO get declared fee period', async () => {
+    expect(await ssvNetworkContract.getDeclaredOperatorFeePeriod()).to.equal(helpers.CONFIG.declareOperatorFeePeriod);
+  });
+
+  it('DAO get execute fee period', async () => {
+    expect(await ssvNetworkContract.getExecuteOperatorFeePeriod()).to.equal(helpers.CONFIG.executeOperatorFeePeriod);
+  });
+  
+  it('DAO increase the fee limit emits "OperatorFeeIncreaseLimitUpdate"', async () => {
+    await expect(ssvNetworkContract.updateOperatorFeeIncreaseLimit(1000
+    )).to.emit(ssvNetworkContract, 'OperatorFeeIncreaseLimitUpdate');
+  });
+
+  it('DAO update the declare fee period emits "DeclareOperatorFeePeriodUpdate"', async () => {
+    await expect(ssvNetworkContract.updateDeclareOperatorFeePeriod(1200
+    )).to.emit(ssvNetworkContract, 'DeclareOperatorFeePeriodUpdate');
+  });
+
+  it('DAO update the execute fee period emits "ExecuteOperatorFeePeriodUpdate"', async () => {
+    await expect(ssvNetworkContract.updateExecuteOperatorFeePeriod(1200
+    )).to.emit(ssvNetworkContract, 'ExecuteOperatorFeePeriodUpdate');
+  });
+
+  it('Not a DAO increase fee limit reverts "caller is not the owner"', async () => {
```

Comment:
Refer to above comments from other tests on how we address "not a dao" or "non dao"